docs/ossm/cert-manager/README.md

```diff
@@ -0,0 +1,200 @@
+
+1. Install [istio-csr](https://cert-manager.io/docs/usage/istio-csr).
+
+   ```sh
+   helm repo add jetstack https://charts.jetstack.io --force-update
+
+   helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+     --install \
+     --namespace cert-manager \
+     --wait \
+     --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
+     --set "volumeMounts[0].name=root-ca" \
+     --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
+     --set "volumes[0].name=root-ca" \
+     --set "volumes[0].secret.secretName=istio-root-ca" \
+     --set "app.istio.namespace=istio-system" \
+     --set "app.istio.revisions={default-v1-23-0}"
+   ```
+
+   Note: If your controlplane namespace is not `istio-system`, you will need to update `app.istio.namespace` to match your controlplane namespace.
+
+2. Install `Istio` controlplane in the `istio-system` namespace.
+
+   ```sh
+   oc apply -f - <<EOF
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: default
+   spec:
+     version: v1.23.0
+     namespace: istio-system
+     updateStrategy:
+       type: RevisionBased
+     values:
+       global:
+         caAddress: cert-manager-istio-csr.cert-manager.svc:443
+       pilot:
+         env:
+           ENABLE_CA_SERVER: "false"
+         volumeMounts:
+           - mountPath: /tmp/var/run/secrets/istiod/tls
+             name: istio-csr-dns-cert
+             readOnly: true
+   EOF
+   ```
+
+3. [Verify your deployment](#verify) is configured correctly.
+
+### Verify
+
+1. Deploy a sample application.
+
+   Create the `sample` namespace.
+
+   ```sh
+   oc create namespace sample
+   ```
+
+   For the `RevisionBased` strategy, label your namespace with `istio.io/rev=default-v1-23-0`.
+
+   ```sh
+   oc label namespace sample istio.io/rev=default-v1-23-0
+   ```
+
+   For the `InPlace` strategy, label your namespace with `istio.io/rev=default`.
+
+   ```sh
+   oc label namespace sample istio.io/rev=default
+   ```
+
+   Deploy the `httpbin` application.
+
+   ```sh
+   oc apply -n sample -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/httpbin/httpbin.yaml
+   ```
+
+2. Ensure httpbin pod is Running.
+
+   ```sh
+   oc get pods -n sample
+   ```
+
+   ```sh
+   NAME                       READY   STATUS    RESTARTS   AGE
+   httpbin-67854dd9b5-b7c2q   2/2     Running   0          110s
+   ```
+
+3. Use `istioctl` to ensure httpbin workload certificate matches what is expected.
+
+   ```sh
+   istioctl proxy-config secret -n sample $(oc get pods -n sample -o jsonpath='{.items..metadata.name}' --selector app=httpbin) -o json | jq -r '.dynamicActiveSecrets[0].secret.tlsCertificate.certificateChain.inlineBytes' | base64 --decode | openssl x509 -text -noout
+   ```
+
+### Upgrades - RevisionBased
+
```

Comment:
Does this mean you can only upgrade if you use the RevisionBased method?

Reply:
No it just means that this section is only relevant if you use `RevisionBased`. You don't have to do anything special for `InPlace`.